tests/cli/test_cli_install.py

```diff
@@ -92,3 +92,28 @@ def test_find_conflicts_called_once(
             *channels,
         )
     assert mocked_find_conflicts.call_count == 3
+
+
+
+@pytest.mark.integration
+def test_emscripten_forge(
+    test_recipes_channel: Path,
+    mocker: MockerFixture,
+    tmp_env: TmpEnvFixture,
+    conda_cli: CondaCLIFixture,
+):
+    mocker.patch("conda.cli.common.confirm_yn", return_value=True)
+
+    with tmp_env() as prefix:
+        stdout, _, _ = conda_cli(
+            "create",
+            "--platform=emscripten-32",
```

Comment:
This is wrong, it should be `emscripten-wasm32`, right?

Reply:
correct =)